### DISCUSSION TITLE
Pay to Turn

### DISCUSSION THREAD

**Discussion by chris-bsnake** [BattlesnakeOfficial/feedback#27] [General]
> Originally posted by **altersaddle** on Slack

Steering is expensive!  Snake health is only reduced if it chooses a move that is not the same as the last move.  When health is reduced , it is by an amount equal to the length of the snake (or function there off).  Developers must weigh the expense of a u-turn or wriggly path, and the cost keeps going up!

**Comment by bvanvugt** [comment]
I guess there's an alternative to this where turning is more expensive. Maybe changing direction on your turn costs additional health.
